types/v3/tech-record/get/hgv/testable/index.d.ts

```diff
@@ -285,3 +274,16 @@ export interface HGVPlates {
   plateReasonForIssue?: null | PlateReasonForIssue;
   plateIssuer?: string | null;
 }
+
+export const enum ApprovalType {
+  NTA = "NTA",
+  ECTA = "ECTA",
+  IVA = "IVA",
+  NSSTA = "NSSTA",
+  ECSSTA = "ECSSTA",
+  GB_WVTA = "GB WVTA",
+  Prov_GB_WVTA = "Prov.GB WVTA",
+  Small_series = "Small series",
```

Comment:
Just wondering if these should also be capitalised for consistency, I can see examples in the code base where other enums are written as "PROV_GB_WVTA".

Reply:
Great spot, thanks mate